### PR TITLE
CPM-603: Use UpsertProductCommand in Enrichment PQB integration tests

### PR DESCRIPTION
**Pull request by cemorin**
**Description (for Contributor and Core Developer)**

Use UpsertProductCommand in Enrichment PQB integration tests

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

File: tests/back/Pim/Enrichment/Integration/PQB/Sorter/Metric/MetricSorterIntegration.php [hunk 1]
```diff
@@ -23,27 +24,15 @@ protected function setUp(): void
...
+            new SetMeasurementValue('a_metric', null, null, '15000', 'WATT')
         ]);
 
         $this->createProduct('product_three', [
-            'values' => [
-                'a_metric' => [
-                    ['data' => ['amount' => '-2.5654', 'unit' => 'KILOWATT'], 'locale' => null, 'scope' => null]
-                ]
-            ]
+            new SetMeasurementValue('a_metric', null, null, '2.5654', 'KILOWATT')
```
Comment (cemorin) [inline tests/back/Pim/Enrichment/Integration/PQB/Sorter/Metric/MetricSorterIntegration.php:35]: The a_metric attribute requires an equal or greater than 0 value

File: tests/back/Pim/Enrichment/Integration/PQB/ProductQueryBuilderIntegration.php [hunk 2]
```diff
@@ -124,84 +131,38 @@ protected function setUp(): void
...
+                new SetFamily('familyA'),
+                new SetFileValue(
+                    'a_file',
+                    null,
+                    null,
+                    $this->getFileInfoKey($this->getFixturePath('akeneo.txt'))
+                ),
+                new SetImageValue('a_localizable_image', null, 'en_US', $this->getFileInfoKey($this->getFixturePath('akeneo.jpg'))),
+                new SetImageValue('a_localizable_image', null, 'fr_FR', $this->getFileInfoKey($this->getFixturePath('akeneo.jpg'))),
+                new SetTextValue('a_regexp', null, null, '10000'),
```
Comment (amigaire) [inline tests/back/Pim/Enrichment/Integration/PQB/ProductQueryBuilderIntegration.php:143]: Why replacing the value from '\w+' to '10000' ?

File: tests/back/Pim/Enrichment/Integration/PQB/ProductQueryBuilderIntegration.php [hunk 3]
```diff
@@ -124,84 +131,38 @@ protected function setUp(): void
...
+                new SetPriceCollectionValue('a_scopable_price', 'ecommerce', null, [
+                    new PriceValue(12, 'EUR'),
+                    new PriceValue(14, 'USD'),
+                ]),
+                new SetTextareaValue('a_localized_and_scopable_text_area', 'ecommerce', 'fr_FR', 'Mon textarea localisé et scopable ecommerce'),
+                //new ClearValue('a_localized_and_scopable_text_area', 'ecommerce', 'en_US'),
```
Comment (amigaire) [inline tests/back/Pim/Enrichment/Integration/PQB/ProductQueryBuilderIntegration.php:149]: To remove

File: tests/back/Pim/Enrichment/Integration/PQB/ProductQueryBuilderIntegration.php (hunk 2 above)
Comment (cemorin) [reply tests/back/Pim/Enrichment/Integration/PQB/ProductQueryBuilderIntegration.php:143]: There were no validation done before, and the 'a_regexp' attribute is a text attribute with a regexp validation, and the value was not the right one (a mistake on the previous test)

**Review by mmetayer — APPROVED** [review]

**Review by nmarniesse — APPROVED** [review]

**Pull request merged**